api/src/controllers/communications/helpers.ts

```diff
@@ -0,0 +1,92 @@
+import { URLSearchParams } from "url";
+import axios from "axios";
+import { selectedCommunications } from "./constants";
+
+const suscribeData = (contactId) => ({
+  ContactId: contactId,
+  SegmentsAndIds: selectedCommunications,
+});
+
+export const getToken = async () => {
+  const keycloakClientSecret =
+    process.env.MAUTIC_SUBSSCRIPTION_API_CLIENT_SECRET;
```

Comment:
To add a env variable, one more pr need to be made in here: https://github.com/bcgov-c/tenant-gitops-platform-registry/tree/main/platform-registry-app to update this env variable in registry manifest.

Reply:
And Add another secret in Openshift namespaces